tools/example.py

```diff
@@ -186,6 +187,12 @@ def get_arguments() -> tuple[argparse.ArgumentParser, argparse.Namespace]:
         type=str,
         help="Update outbound WebSocket (Gen2/3) and exit",
     )
+    parser.add_argument(
+        "--listen_ip_address",
+        "-lip",
+        type=str,
+        help="Listen ip address for incoming CoAP packets",
```

Comment:
```suggestion
        type=str,
        nargs='*',
        default=None,        
        help="Listen ip address for incoming CoAP packets",
```

With `nargs='*'` output will be a list and default can be set to `None` if no input
https://docs.python.org/3/library/argparse.html#nargs

Example output of the above with different input
`example.py --ip 1.2.2.3`  --> `args.listen_ip_address=None`
`example.py --ip 1.2.2.3 -lip 1.2.3.4` -->  `args.listen_ip_address=['1.2.3.4']`
`example.py --ip 1.2.2.3 -lip 1.2.3.4 5.6.7.8` --> `args.listen_ip_address=['1.2.3.4', '5.6.7.8']`

Reply:
Nice, thx !